KSSAdditionalFile

```diff
@@ -0,0 +1 @@
+This is another demo file
```

Comment:
Change the statement and add a sensible paragraph

Reply:
Updated my file accordingly